impeller/entity/geometry/point_field_geometry.cc

```diff
@@ -185,8 +185,8 @@ GeometryResult PointFieldGeometry::GetPositionBufferGPU(
   }
 
   if (texture_coverage.has_value() && effect_transform.has_value()) {
-    BufferView geometry_uv_buffer =
-        host_buffer.Emplace(nullptr, total * sizeof(Vector4), alignof(Vector4));
+    BufferView geometry_uv_buffer = host_buffer.Emplace(
+        nullptr, total * sizeof(Vector4), DefaultUniformAlignment());
```

Comment:
Ditto about the max.

Reply:
Done in https://github.com/flutter/engine/pull/51341